oat-sa/bundle-lti1p3
# fix: convert exception message to html for response

# [TR-2645](https://oat-sa.atlassian.net/browse/TR-2645)

## Goal
Avoid XSS attacks and fix psalm errors

## Changelog

- feat: new ExceptionResponse to convert exception message to html
- chore: add return types in some methods

## How to test
1. Set up TAO as an LTI _1.1_ platform
2. Set up TAO Advance as a tool
3. Go to path: /lti1p3/oidc/initiation?iss=%3Cimg+src=https://localhost+onerror=alert(1)%3E&login_hint=
4. See error message in html format
